diff --git a/node_modules/cacheable-lookup/source/index.js b/node_modules/cacheable-lookup/source/index.js
index 21f731e..41bab01 100755
--- a/node_modules/cacheable-lookup/source/index.js
+++ b/node_modules/cacheable-lookup/source/index.js
@@ -7,7 +7,7 @@ const {
 		Resolver: AsyncResolver
 	},
 	lookup: dnsLookup
-} = require('dns');
+} = require('@i2labs/dns');
 const {promisify} = require('util');
 const os = require('os');
 
