diff --git a/node_modules/http2-wrapper/source/client-request.js b/node_modules/http2-wrapper/source/client-request.js
index b712967..c4c47ce 100644
--- a/node_modules/http2-wrapper/source/client-request.js
+++ b/node_modules/http2-wrapper/source/client-request.js
@@ -15,11 +15,18 @@ const {
 	ERR_INVALID_CHAR
 } = require('./utils/errors');
 
+http2.constants = {
+  HTTP2_HEADER_STATUS: ':status',
+  HTTP2_HEADER_METHOD: ':method',
+  HTTP2_HEADER_PATH: ':path',
+  HTTP2_METHOD_CONNECT: 'CONNECT',
+}
+
 const {
 	HTTP2_HEADER_STATUS,
 	HTTP2_HEADER_METHOD,
 	HTTP2_HEADER_PATH,
-	HTTP2_METHOD_CONNECT
+	HTTP2_METHOD_CONNECT,
 } = http2.constants;
 
 const kHeaders = Symbol('headers');
